modules/odf/main.tf

```diff
@@ -2,16 +2,48 @@
 # Install ODF on the cluster
 ##################################
 
+locals {
+  installation_content = templatefile("${path.module}/templates/install_odf.yaml.tmpl", {
+    #monSize = var.monSize,
+    #monStorageClassName = var.monStorageClassName,
```

Comment:
Need logic to check for version.  If 4.7, include `monSize` and `monStorageClassName`